### ISSUE TITLE
not seeing successfull transaction in copay

### ISSUE THREAD

**Issue by ozf5xo5e6abefvbsjk7f** [bitpay/copay#6227]
Hi folks!! first of all, awesome work.. 
Second.. 
I imported my wallet into another wallet app (two actually) i can see a couple of transactions there that i can not see on copay.
at the beginning i though I did not refresh it enough, now im not able to find that rescan for funds option in the android app.
then i thought it was a matter of time for it to appear.. 
now the transaction has a lot of confirmations, but i still cant see in copay... i dunno why.

today I was going to stop using copay, but a friend suggested to file a bug instead. and here i am.

**Comment by JDonadio** [comment]
Hi @ozf5xo5e6abefvbsjk7f, please explain what was exactly what you did so I can check your case further.

**Comment by ozf5xo5e6abefvbsjk7f** [comment]
i created a wallet on copay, have had it for a while now, put a few coins in it, and then I decided to try another HD wallet using the same BIP 32 thing. 
I installed the other wallet on my computer, and imported my wallet using the 12 words..  made a few transactions with the new wallet app, and then i went to check on copay whether I could see the transactions in copay. turns out i couldnt..

I was ok for a while, until it started to bother me.

Last night I installed a new wallet up on my phone, also to test it, and imported this wallet to see if it was able to see that particular transaction as well. And it could.

More a few coins around and it workrd like a charm, how ever it moved the unspent funds from the transaction that copay can not see.. so the transaction history on copay remains the same,

Now, i dont know if this is important, but I chose blockchain.info as a block explorer on the latest wallet. perhaps it is a problem onthe bitpay api, and not in copay itself.

any ideas?
